### PR TITLE
Support running RayFed job in Ray client mode.

### LINKED ISSUE #75: [Feature] Support running RayFed job in Ray client mode.

### PR DESCRIPTION
**Pull request by jovany-wang**
Close #75

### CODE REVIEW

**Comment by jovany-wang** [comment]
TESTS should be added.

File: fed/api.py [hunk 1]
```diff
@@ -275,6 +275,7 @@ class FedRemoteClass:
     def __init__(self, func_or_class) -> None:
         self._party = None
         self._cls = func_or_class
+        # self._client_actor_handle_class = ray.
```
Comment (fengsp) [inline fed/api.py:278]: clean this line?

**Review by fengsp — COMMENTED** [review]
LGTM

**Comment by jovany-wang** [comment]
Tests was added.

File: fed/api.py (hunk 1 above)
Comment (jovany-wang) [reply fed/api.py]: DONE

**Pull request merged**